spec/core_functions/color/hsla/four_args/special_functions.hrx

```diff
@@ -256,3 +256,63 @@ a {
   b: hsla(1, 2%, 3%, max(0.4));
 }
 
+<===>
+================================================================================
+<===> clamp/arg_1/options.yml
```

Comment:
Would it make sense to use a single `clamp/options.yml` file instead of separately disabling each test for libsass?